### ISSUE TITLE
Questions about the model of match pyramid.

### ISSUE THREAD

**Issue by yananchen1989** [faneshion/MatchZoo#97]
I want to realize the match matrix (2-D tensor as image) which is the Dot / Cosine / Gaussian Kernel / Indicator from the query and doc embedding list.
I only find the Dot operation in the codes, which can be seen in `/matchzoo/models/matchpyramid.py`
`cross = Dot(axes=[2, 2], normalize=False)([q_embed, d_embed])`
The other ways like Cosine / Gaussian Kernel operation has not been found. 
Is there any sample codes ? Especially in tensorflow ?
Thanks.

**Comment by bwanglzu** [comment]
See [Keras Documentation](https://keras.io/layers/merge/#dot)

The argument `normalize=False` means dot product between `q_embed` and `d_embed`. Meanwhile, `normalize=True` means `cosine similarity`.

I thought you were asking questions on distance metrics, but I have no idea how it's related to Gaussian Kernel.

**Comment by bwanglzu** [comment]
I'll close this issue for now, if you have further question please feel free to re-open it or create a new one.
